.protolint.yml

```diff
@@ -39,7 +39,7 @@ lint:
     add:
       - FIELD_NAMES_LOWER_SNAKE_CASE
       - MESSAGE_NAMES_UPPER_CAMEL_CASE
-      - MAX_LINE_LENGTH
+      # - MAX_LINE_LENGTH
```

Comment:
Please bring this back.

Reply:
`buf format` deos not limit line_length this way. If we set `MAX_LINE_LENGTH` here, it will be conflict with `buf format`

Reply:
<img width="2556" alt="Screenshot 2024-12-09 at 10 43 25" src="https://github.com/user-attachments/assets/d623176f-2deb-4df1-b0e6-b7d3c9cab4a1">
The left side is `buf` way, and the right side is `clang` way. `buf` way looks better.

Reply:
I see, but often we (as developers) end up making the line too long. That adds difficulty in reading it. If there is no option, please proceed.